qt/python/mantidqt/mantidqt/plotting/sample_shape.py

```diff
@@ -241,9 +244,8 @@ def get_component_shape_from_workspace(workspace_with_components, component_inde
 def plot_sample_only(workspace, figure):
     axes = figure.gca()
     # get shape and mesh vertices
-    shape = get_valid_sample_shape_from_workspace(workspace)
-    if shape:
-        mesh = shape.getMesh()
+    mesh = get_valid_sample_mesh_from_workspace(workspace)
+    if mesh is not None:
```

Comment:
I think `if mesh` is equivalent to `if mesh is not None` but okay if you just want to be explicit about it 👍